packages/node_modules/@webex/helper-image/src/index.js

```diff
@@ -43,21 +43,27 @@ export function updateImageOrientation(file, options = {}) {
 * @returns {Promise<ExifImage>}
 */
 export function readExifData(file, buf) {
-  return new Promise((resolve) => {
+  return new Promise(async (resolve) => {
     // For avatar images the file.type is set as image/jpeg, however for images shared in an activity file.mimeType is set as image/jpeg. Handling both conditions.
-    if (file && (file.type === 'image/jpeg' || file.mimeType === 'image/jpeg')) {
-      /* eslint-disable no-new */
-      new ExifImage({image: buf}, (error, exifData) => {
-        if (!error && exifData) {
-          file.orientation = exifData.image.Orientation;
-          file.exifWidth = exifData.exif.ExifImageWidth;
-          file.exifHeight = exifData.exif.ExifImageHeight;
-          if (file.image) {
-            file.image.orientation = exifData.image.Orientation;
-          }
-        }
-        resolve(buf);
-      });
+    if (
+      file &&
+      (file.type === 'image/jpeg' || file.mimeType === 'image/jpeg')
+    ) {
+      const {
+        Orientation,
+        ExifImageHeight,
+        ExifImageWidth
+      } = await parse(buf, {translateValues: false});
```

Comment:
in this case, if an error is thrown here, the promise returned by `readExifData` *should* reject, so the original error validation does not appear to be necessary any longer.